src/graphorm/index.ts

```diff
@@ -422,6 +431,7 @@ const obj = new GraphORM({
     },
   },
   UserPost: {
+    requiredColumns: ['votedAt'],
```

Comment:
Why is it required here as seen it's an optional field in the entity

Reply:
From what I understand its not a matter of optional but just so that this column is always added to select db query no matter what user requests from gql query. `Upvote.createdAt` was like that so this is why I also added `votedAt`. cc: @idoshamun is my reasoning ok or do we use `requiredColumns` for something else too?

Reply:
Ah yes sorry think I'm thinking of the wrong place.
Here it indeed says it will always be added.

Reply:
Yeah, but valid comments because I was wondering if it is needed in the first place.

Reply:
Cool, since we order by `votedAt` then it should be here ✅